wrapanapi/systems/scvmm.py

```diff
@@ -229,6 +230,40 @@ def enable_virtual_services(self):
         )
         self.system.run_script(script)
 
+    def create_snapshot(self, check_type="Standard"):
```

Comment:
While this is an appropriate name for the function, in order to maintain consistency with the other system's VM objects, this method should be called `mark_as_template`, and should take kwargs.

Reply:
There was confusion about what this function is doing, cleared up offline. This function is for creating a snapshot/checkpoint, not for creating a template from a VM.